book/src/bridge/attributes.md

```diff
@@ -26,6 +28,16 @@ The `#[cxx_name = "..."]` attribute replaces the name that C++ should use for th
 
 The `#[rust_name = "..."]` attribute replaces the name that Rust should use for this item.
 
+For QProperties, a CXX or Rust name can be provided inside the macro like so
+
+`#[qproperty(T, name, cxx_name = "MyName", rust_name = "my_name")]`
+
+If no `cxx_name` or `#[cxx_name = "..."]` is provided, C++ will use the provided rust name, except for capitalising the first letter for autogenerated functions
+e.g. a property called my_property will have the auto generated getter getMy_property in C++
+
+> Note that in some cases `snake_case` conversions may occur for generated functions in Rust (eg `on_<signal>`).
+> This will use the cxx_name of the signal however, so a signal called `MySignal`will have the generated function on_MySignal
```

Comment:
@LeonMatthesKDAB what should we do about this text now that you've modified the book? Some parts of it are relevant maybe, but are they described in the property section better i wonder? :thinking:

Reply:
I think we should just keep the example of how to specify `rust_name` and `cxx_name` on qproperty (e.g. the first two lines) and then link to the property section for details.

Reply:
done